### DISCUSSION TITLE
Format of Exploration Task Mapping

### DISCUSSION THREAD

**Discussion by Aanal2901** [larics/icuas22_competition#27] [Q&A]
@aivano @AnaBatinovic 
For difficulty 2, task 1 we are supposed to map our environment. Do we have to provide/submit this map? If yes, what is the format and what is the extent of mapping?

**Comment by fpetric** [comment]
@Aanal2901 thanks for the question. Let me jump in to answer, no, you do not need to submit the map. We were discussing evaluating the quality of the map at some point, but decided that it would be difficult to make a fair evaluation, so the only criteria for that part is speed (and not crashing into obstacles). :)

**Reply by afaroo01** [reply]
Hi @aivano @AnaBatinovic , Another question, I would like to know if we can use 2D SLAM mapping approach for exploration in difficulty level 2. As I read for level 1, we can ask for a prior octomap map created by 3D SLAM, I suppose. So does it make any difference in scoring and will be considered valid using 2D SLAM for level 2?

**Reply by fpetric** [reply]
In short, you can use whatever approach you deem the best. If all of the obstacles are the same height (which they do not have to be) you may not find the best way to go through Zone 2. The challenge was envisioned as 3D exploration and obstacle avoidance, so we will try to make the evaluation scenarios in a way in which there is actual benefit of using 3D instead of 2D. But if 2D works for you, you will not be penalized for not using 3D techniques.
